docker-build-x86-64/Dockerfile

```diff
@@ -0,0 +1,34 @@
+from phusion/holy-build-box-64:latest
+
+# Install graphical libraries (some packages, such as Matplotlib, require them
+#    at test time)
+run yes | yum install libX11-devel libSM libXrender
+run yes | yum install openssh-clients chrpath
+run yes | yum install vim-enhanced vim-common nano
+# We provide our own python.  Remove HBB's version.
+run rm /hbb/bin/python
```

Comment:
Good idea, but no. That python is part of what hbb adds, not something that
yum can manage.

On Thu, Jan 14, 2016, 17:02 Ryan Grout <notifications@github.com> wrote:

> In docker-build-x86-64/Dockerfile
> <https://github.com/ContinuumIO/docker-images/pull/20#discussion_r49799251>
> :
>
> > @@ -0,0 +1,34 @@
> > +from phusion/holy-build-box-64:latest
> > +
> > +# Install graphical libraries (some packages, such as Matplotlib, require them
> > +#    at test time)
> > +run yes | yum install libX11-devel libSM libXrender
> > +run yes | yum install openssh-clients chrpath
> > +run yes | yum install vim-enhanced vim-common nano
> > +# We provide our own python.  Remove HBB's version.
> > +run rm /hbb/bin/python
>
> Wouldn't it be better to remove python via yum?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ContinuumIO/docker-images/pull/20/files#r49799251>.
>